metagenomics_samplesheet_generator.ipynb

```diff
@@ -152,7 +152,7 @@
        "      <td>33-A1</td>\n",
        "      <td>A</td>\n",
        "      <td>1</td>\n",
-       "      <td>False</td>\n",
+       "      <td>True</td>\n",
```

Comment:
It's not clear to me why some values that were previously False have been changed to True in this PR. It might be good to describe that in the comments.